CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# How to contribute
+
+CassKop is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the
+conventions on commit message formatting, contact points for developers, and other resources to help get contributions
+into CassKop.
+
+## Email and Chat
+
```

Comment:
There is no email. Should we keep only Slack ? I think it makes sense

Reply:
You can add the e-mail : prj.casskop.support@list.orangeportails.net